src/3box.js

```diff
@@ -342,6 +351,12 @@ class Box {
       } catch (e) {
         delete this.spaces[name]
       }
+      const syncSpace = async () => {
+        const numEntries = this.spacesPubSubMessages[name] ? this.spacesPubSubMessages[name].numEntries : null
```

Comment:
Wouldn't it be easier to send the number of msgs to the open function rather than to move code into the 3box main module?

Reply:
oh yeah, if this is a minimal code change for now, yeah I should keep this over in the open function  👍